### PR TITLE
Fix getAllLaunches Variable Mismatch

### PR DESCRIPTION
**Pull request by alaneparisot**
Replace `result` with `response`.

### CODE REVIEW

File: docs/source/tutorial/data-source.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ _src/datasources/launch.js_
 ```js
 async getAllLaunches() {
   const response = await this.get('launches');
-  return Array.isArray(result)
+  return Array.isArray(response)
```
Comment (abernix) [inline docs/source/tutorial/data-source.md]: I think both of these variables actually need to be changed to `res` in order to match up with [the `fullstack-tutorial` repository's](https://github.com/apollographql/fullstack-tutorial) "final" state, as seen in [this commit](https://github.com/apollographql/fullstack-tutorial/blob/7948b85d747300fd4d1596b6173b9ee783431c75/final/server/src/datasources/launch.js#L28-L33).

File: docs/source/tutorial/data-source.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ _src/datasources/launch.js_
 ```js
 async getAllLaunches() {
   const response = await this.get('launches');
```
Comment (abernix) [inline docs/source/tutorial/data-source.md]:
```suggestion
  const res = await this.get('launches');
```

File: docs/source/tutorial/data-source.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ _src/datasources/launch.js_
...
-  return Array.isArray(result)
+  return Array.isArray(response)
     ? response.map(launch => this.launchReducer(launch))
```
Comment (abernix) [inline docs/source/tutorial/data-source.md]:
```suggestion
    ? res.map(launch => this.launchReducer(launch))
```

File: docs/source/tutorial/data-source.md (hunk 1 above)
Comment (abernix) [inline docs/source/tutorial/data-source.md]:
```suggestion
  return Array.isArray(res)
```

Comment (evans) [reply docs/source/tutorial/data-source.md]: Let's continue with `response` and change the final state of the `fullstack-tutorial`. I think #294 makes a very good point about `res` vs `response`

Comment (abernix) [reply docs/source/tutorial/data-source.md]: I also agree with, @cdimitroulas!

Let’s make sure we update the tutorial!

Comment (cdimitroulas) [reply docs/source/tutorial/data-source.md]: yay for clearer names :hugs:

**Pull request merged**